package.json

```
@@ -16,6 +16,7 @@
   "scripts": {
     "build": "tsc",
     "start": "node ./build/index.js",
+    "start:ts": "node --loader ts-node/esm src/index.ts",
```

Let's just unify these together
```suggestion
    "dev": "node --loader ts-node/esm src/index.ts -V",
```
Then we can just make the `debug` as follows (since it's still useful to have an option to have)
```
"debug": "npm run dev -- --inspect"
```

---

I have changed. please review again.

"debug": "npm run dev -- --inspect"
This modification doesn't work, --inspect must be placed after node, therefore there's no need to change this debug script.